### PR TITLE
Local Booster Catalog Testing

### LINKED ISSUE #389: Document process for updating catalog for local minishift testing
When adding a new runtime or booster, developers need to know how to update the local catalog for testing with minishift. The contrib guide should contain this information.

### PR DESCRIPTION
**Pull request by tradej**
Resolves #389.

### CODE REVIEW

**Comment by rhoads-zach** [comment]
[test]

**Comment by tradej** [comment]
Ready for review.

**Comment by tradej** [comment]
@lholmquist, can you please have a look too?

**Comment by inoxx03** [comment]
LGTM:)

**Comment by tradej** [comment]
@gastaldi, can you please re-review? There have been substantial changes. Thank you.

File: docs/topics/assembly-updating-booster-catalog-for-local-minishift-testing.adoc [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+= Updating Booster Catalog for Local {OpenShiftLocal} Testing
+
+One of important life cycle stages of a booster is testing it in a booster catalog.
```
Comment (gastaldi) [inline docs/topics/assembly-updating-booster-catalog-for-local-minishift-testing.adoc]: One of `the`

**Comment by inoxx03** [comment]
merging

**Comment by inoxx03** [comment]
merged: a5124987cca2f1316806cf414931b3a3816bd7c3

**Pull request closed without merging**